exercises/perfect-numbers/example.el

```diff
@@ -3,26 +3,27 @@
 ;;; Commentary:
 
 ;;; Code:
-
-(require 'cl-lib)
-
-
-(defun divisors (n)
-  (cl-loop for i from 1 to (truncate (sqrt n))
```

Comment:
If n were 14 then this would range up to 3.  7 is also a factor of 14, so this method is broken.